API/tests/createTask_test.go

```
@@ -0,0 +1,205 @@
+
+	"llio-api/database"
+	"llio-api/models/DAOs"
+	"llio-api/models/DTOs"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestCreateTask est la fonction de test pour le contrôleur CreateTask.
+func TestCreateTask(t *testing.T) {
+
+	router, w := setupTestRouter()
```

golangci-lint (1.62.2)

ineffectual assignment to w

(ineffassign)